### ISSUE TITLE
Color contrast ratio for highlighted menu item is 2.704:1 which is less than required ratio 3:1.

### ISSUE THREAD

**Issue by BhanuPrakashGit** [dotnet/wpf#6481]
**GitHubTags**: #A11yMAS;#A11ySev3;#A11yTCS;#DesktopApp;#Win10;#BM-.NetCore(WPF)-Win32-July2021;#.NET Core;#Nonbenchmark; WCAG1.4.11;

### Environment Details:
Application Name: Accessibility Insights for Windows
Windows Version: Win10
Application Version: Version 1.1.1741.1

### Repro Steps:

1. Open WPF-Samples-Master.
2. Open ControlsAndLayout.exe file.
3. Tab to "File" button and hit enter.
4. Observe color contrast ratio for highlighted menu item is 2.704:1 which is less than required ratio 3:1.

### Actual:
Color contrast ratio for highlighted menu item is 2.704:1 which is less than required ratio 3:1.

### Expected:
Color contrast ratio for highlighted menu item should be equal or greater than required ratio 3:1.

### User Impact:
Low vision users will not be able to see the current highlighted menu item.

[sample-app.zip](https://github.com/dotnet/wpf/files/8553045/sample-app.zip)

https://user-images.githubusercontent.com/104351566/165056622-df88ea6a-ea87-429d-9fc8-0e5215c24b14.mp4

**Comment by BhanuPrakashGit** [comment]
Verified the bug in the latest env as bug is fixed with a workaround by bolding the selected menu item's text.
